tests/Console/Command/SelfUpdateCommandTest.php

```diff
@@ -47,8 +47,8 @@ protected function setUp(): void
 
         file_put_contents($this->getToolPath(), 'Current PHP CS Fixer.');
 
-        file_put_contents("{$this->root->url()}/{$this->getNewMinorReleaseVersion()}.phar", 'New minor version of PHP CS Fixer.');
-        file_put_contents("{$this->root->url()}/{$this->getNewMajorReleaseVersion()}.phar", 'New major version of PHP CS Fixer.');
+        file_put_contents($this->root->url().'/'.$this->getNewMinorReleaseVersion().'.phar', 'New minor version of PHP CS Fixer.');
```

Comment:
why the change? what's wrong with original code?

Reply:
I am noob: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7136/commits/f24ea6ad59308c6d025bddb06ceb566f9ddb5482

Reply:
why changing interpolation to concatenation?

Reply:
@keradus because you can't interpolate call to static method.